app/src/main/java/com/infomaniak/drive/ui/login/LoginActivity.kt

```diff
@@ -198,6 +203,12 @@ class LoginActivity : AppCompatActivity() {
         signInButton.isEnabled = true
     }
 
+    private fun handleHelpShortcut() {
+        navigationArgs?.isHelpShortcutPressed?.let { isHelpShortcutPressed ->
+            if (isHelpShortcutPressed) openSupport()
+        }
```

Comment:
You can online this
also I don't think it is necessary to add the let, this does it well

```suggestion
       if (navigationArgs?.isHelpShortcutPressed == true) openSupport()
```